src/schema/rules/tabular_data/derivatives/functional_derivatives.yaml

```
@@ -0,0 +1,44 @@
+---
```

`timeseries.tsv` can contain a superset of what's in these other files.

```suggestion
---
GeneralTimeSeries:
  selectors:
  - dataset.DatasetType == "derivative"
  - suffix == 'timeseries'
  columns:
  - trans_x: optional
  - trans_y: optional
  - trans_z: optional
  - rot_x: optional
  - rot_y: optional
  - rot_z: optional
  - framewise_displacement: optional
  - rmsd: optional
  - rms: optional
  - non_steady_state_x: optional
  - dvars: optional
  - std_dvars: optional
  - cosine_x: optional
  - legendre: optional
  - compcor: optional
  - melodic_x: optional
  additional_columns: allowed
```